docs/resource-policy/policy/topology-aware.md

```diff
@@ -443,7 +443,7 @@ metadata:
     prefer-reserved-cpus.resource-policy.nri.io/container.special: "false"
 ```
 
-## Allowing or denying mount/device paths via annotations
+## Controlling Topology Hints Via Annotations
 
```

Comment:
this probably should not be part of this PR

Reply:
Argh, true.